cli/dts/lib.deno.unstable.d.ts

```diff
@@ -685,53 +613,53 @@ declare namespace Deno {
    *
    * NOTE: This functionality is not yet implemented on Windows.
    */
-  export function signal(signo: number): SignalStream;
+  export function signal(signo: string): SignalStream;
 
   /** **UNSTABLE**: new API, yet to be vetted. */
   export const signals: {
     /** Returns the stream of SIGALRM signals.
      *
-     * This method is the shorthand for `Deno.signal(Deno.Signal.SIGALRM)`. */
+     * This method is the shorthand for `Deno.signal("SIGALRM")`. */
     alarm: () => SignalStream;
```

Comment:
So `Deno.signals.alarm()` is shorthand for `Deno.signal("SIGALRM")` ... seems useless. We should remove the entire `Deno.signals` namespace IMO (cc @kt3k)

Reply:
I'm in favor - it's just syntactic sugar

Reply:
removed